src/tests/copy_count_tracker.hpp

```diff
@@ -54,8 +54,11 @@ class copy_count_tracker
     {
         return rpp::source::create<copy_count_tracker>([this, count](const auto& sub)
         {
-            for (size_t i = 0; i < count; ++i)
+            for (size_t i = 0; i < count; ++i) {
+                if (!sub.is_subscribed()) { break; }
+
                 sub.on_next(*this);
+            }
```

Comment:
Same as below
```suggestion
            for (size_t i = 0; i < count && sub.is_subscribed(); ++i) 
                sub.on_next(*this);
```

Reply:
Addressed in the latest commit (overriden)